hub-config/tasks.json

```diff
@@ -115,9 +115,9 @@
                                     "moderate",
                                     "high",
                                     "very high"
-                                ],
-                                "optional": null
+                                ]
                             },
+                            "is_required": true,
```

Comment:
I'm assuming here that because all output_type_ids were required, that should translate to an overall required output type, which is conditional on whether a given model task is being submitted to. Please correct me if I'm wrong. The response also relates to outtypes marked as required below.